### PR TITLE
Transport underglow implementation to default map

### PR DESCRIPTION
**Pull request by Biacco42**
Underglow support for Ergo42 default keymap

### CODE REVIEW

File: keyboards/ergo42/keymaps/default-underglow/rules.mk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/ergo42/keymaps/default-underglow/rules.mk]: This block isn't needed, you should only need the `RGBLIGHT_ENABLE = yes` line in this file.

The rest is a holdover from the old makefile system, that just hasn't been cleaned up yet, and isn't necessary

File: keyboards/ergo42/keymaps/default-underglow/keymap.c [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      #endif
+      break;
+    case TAP_ANIM:
+      if (record->event.pressed) {
+        isTapAnim = !isTapAnim;
+      }
+      break;
+  }
+
+  if (isTapAnim) {
```
Comment (drashna) [inline keyboards/ergo42/keymaps/default-underglow/keymap.c]: For this stuff, if you wanted a "funner" effect, I have a "twinkling" type effect here:

https://github.com/qmk/qmk_firmware/blob/master/users/drashna/rgb_stuff.c#L108-L253

What this does, is each key you hit causes a single LED to "light up" and twinkle basically.   It's a bit complex, but if you wanted to use that instead, then feel free!

**Comment by Biacco42** [comment]
Thanks for your comment. I removed needless part.
I will check your twinkling effect!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**